spec/xmlspec-ttml1.xsl

```diff
@@ -612,6 +608,44 @@ width: 100%;
   </xsl:choose>
 </xsl:template>
 
+<!-- implreploc: implementation report for this spec -->
+<!-- called in a <dl> context from header -->
+<xsl:template match="implreploc">
+  <xsl:choose>
+    <xsl:when test="count(loc) &gt; 1">
+      <xsl:for-each select="loc">
+        <dt>
+          <xsl:apply-templates select="node()"/>
+        </dt>
+        <dd>
+          <a href="{@href}">
+            <xsl:value-of select="@href"/>
+          </a>
+        </dd>
+      </xsl:for-each>
+    </xsl:when>
+    <xsl:otherwise>
+      <dt>Implementation report:</dt>
+      <dd>
+        <xsl:apply-templates/>
+      </dd>
+    </xsl:otherwise>
+  </xsl:choose>
+</xsl:template>
+
+<!-- ghrepoloc: participate section -->
+<!-- called in a <dl> content from header -->
+<xsl:template match="ghrepoloc">
+  <xsl:apply-templates/>
+</xsl:template>
+<xsl:template match="ghloc">
+  <dt>Participate:</dt>
+  <dd><a href="https://github.com/{@repo}/">GitHub <xsl:value-of select="@repo"/></a></dd>
+  <dd><a href="https://github.com/{@repo}/issues/">File a bug</a></dd>
```

Comment:
This goes to the issues list not the "create an issue" page - is that intentional?

Reply:
lines are from what respec inserts (e.g. https://w3c.github.io/mediacapture-main/), and I'm not sure what was the original intent...
Actually, bikeshed inserts a link pointing issues/new, so might be better to raise this to respec also?

Reply:
Makes sense to have consistency, yes. Anyway I think we can go straight for issues/new since the text says "File a bug".